cegs_portal/search/views/v1/search.py

```diff
@@ -455,6 +464,17 @@ def request_options(self, request):
         return options
 
     def get(self, request, options, data):
+        if request.headers.get("HX-Target"):
+            return render(
+                request,
+                self.partial,
```

Comment:
I think you can hardcode `"search/v1/partials/_feature_sig_reg_effects_table.html"` here. It's a little weird to use "self.partial" because the value of the other option (self.target) is also a partial.